File: 2017 COMLAN/RangeValueMonitor/src/matlab/foobar.m
function foobar()
    
    foo();

end

function a = foo()
    a = 0;
    
    for i = 0:5
        a = a + bar();
    end
    
	a = libcall(a);
	
end


function result = bar() 
    result = 1.0;
end

File: 2017 COMLAN/RangeValueMonitor/src/c/foobar.c
#include "lib/lib.h"

double bar() {
    return 1.0;
}

double foo() {
    double a = 0;
    
    for(int i=0; i<1000; i++) {
        a += bar();
    }

	a = lib_call(a);
    
    return a;
}

int main() {
    
    foo();

}